kyuubi-server/src/main/scala/org/apache/kyuubi/events/StatementEvent.scala

```diff
@@ -0,0 +1,66 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kyuubi.events
+
+import org.apache.kyuubi.Utils
+import org.apache.kyuubi.operation.ExecuteStatement
+import org.apache.kyuubi.operation.OperationState.OperationState
+
+/**
+ *
+ * @param user: who connect to kyuubi server
+ * @param statementId: the identifier of operationHandler
+ * @param statement: the sql that you execute
+ * @param remoteIp: the ip of user
+ * @param sessionId: the identifier of a session
+ * @param createTime: the create time of this statement
+ * @param state: store each state that the sql has
+ * @param stateTime: the time that the sql's state change
+ * @param exception: caught exception if have
+ */
+case class StatementEvent(
```

Comment:
This enigne type is conflict with engine's `StatementEvent` if we write into same path.

Reply:
> This event type is conflict with engine's `StatementEvent` if we write into same path.

`StatementEvent` is a path too, they have differnet file name under `statement`. 
And add a check for this case. 
```
ll /var/folders/xl/jv1jvw6s6hv42ht1ty6gnk100000gn/T/kyuubi-bb4a5f34-451c-4783-9f50-1906889386cf/statement/day=20210824/
-rw-rw----  1 *  staff  9472 Aug 24 16:27 local-1629793620908.json
-rw-rw----  1 *  staff  3588 Aug 24 16:27 server-172.20.144.37.json
```

Reply:
Since they have different schema, if we want to create table on that path, how can we read them together ? I think we should use different event type for this case so that we can create tables for them.

Reply:
> Since ... I think we should use different event type for this case so that we can create tables for them.

Added prifix for StatementEvents.